tyk/README.md

```diff
@@ -0,0 +1,145 @@
+# Agent Check: tyk
+
+## Overview
+
+![tyk](https://tyk.io/wp-content/uploads/2021/07/tyk_logo_no_bg.png)
+
+![tyk-in-action](./images/tyk-in-lab.png?raw=true "Tyk's Lab")
+
+This check collects metrics from [Tyk-gateway][https://github.com/TykTechnologies/tyk].
```

Comment:
All links should be numbered notation, for our translation pipelines to work properly. It looks like you do have some at the end of the file, please update the rest of them to follow that format.